storage/pebble/registers.go

```diff
@@ -59,12 +59,9 @@ func (s *Registers) Get(
 			fmt.Sprintf("height %d not indexed, indexed range is [%d-%d]", height, s.firstHeight, latestHeight),
 		)
 	}
-	iter, err := s.db.NewIter(&pebble.IterOptions{
+	iter := s.db.NewIter(&pebble.IterOptions{
```

Comment:
new version of Pebble won't error on Iterator creation and returns error on operations